elliott/elliottlib/shipment_utils.py

```
@@ -112,3 +104,55 @@ def set_jira_bug_ids(release_notes: ReleaseNotes, bug_ids: Iterable[str]):
+    """
+    parsed_url = urlparse(shipment_data_url)
+    project_path = parsed_url.path.strip("/")
+
+    gl = GitLabClient.from_url(shipment_data_url)
+    open_mrs = gl.list_merge_requests(project_path, state="opened")
+
+    bug_ids: set[str] = set()
+    for mr in open_mrs:
+        try:
+            shipment_configs = get_shipment_configs_from_mr(mr.web_url)
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)